addons/stock_account/tests/test_stockvaluationlayer.py

```diff
@@ -261,6 +261,44 @@ def test_change_in_past_increase_dropship_1(self):
         self.assertEqual(self.product1.value_svl, 0)
         self.assertEqual(self.product1.quantity_svl, 0)
 
+    def test_empty_stock_move_valorisation(self):
```

Comment:
Add it in https://docs.google.com/spreadsheets/d/1igft1hDrT6y8CApETMtKnXP1Up8CkjgNvkZhRqruBN8/edit?usp=sharing